docs/pages/deploy/config-edit.en-US.mdx

```diff
@@ -0,0 +1,100 @@
+
+## Step One: Create a Project
+
+Create a Project in [Zeabur Dash](https://dash.zeabur.com/).
+
+## Step Two: Create Services
+
+Create Prebuilt Services in the aforementioned Project , with service names namely nginx and vaultwarden.
+
+![Create Ngnix Service](/deploy/config-edit/create-nginx-service.en-US.png)
+
+![Create Vaultwarden Service](/deploy/config-edit/create-vultwarden-service.en-US.png)
+
+## Step Three: Edit Nginx Service Configuration File
+
+1. In the Services column, click `Settings` tab for the nginx service, click the drop-down button to find the `Configs` section, and click the `Open Config Editor` button to enter the configuration file editor.
+
+![Open Configuration File Editor](/deploy/config-edit/open-config-editor.en-US.png)
+
+2. In the opened window, click the `Add Config file` button, to add a configuration file. In the popped-up window, input the path and name of the configuration file in the `Config Path` input box. Here, we input `/etc/nginx/nginx.conf` and click the `Save Config` button.
+
+![Add Configuration File](/deploy/config-edit/create-new-config.en-US.png)
+
+<Callout>
+    The `Config Path` above requires the absolute path of the configuration file, please configure in line with the actual situation.
+</Callout>
+
+3. Depending on the requirements, edit the content of the configuration file. Here, we add a `server` configuration for reverse-proxying the vaultwarden service.
+
+```nginx
+worker_processes  4;
+error_log  stderr;
+worker_rlimit_nofile 8192;
+
+events {}
+
+http {
+  default_type application/octet-stream;
+  log_format   main '$remote_addr - $remote_user [$time_local]  $status '
+    '"$request" $body_bytes_sent "$http_referer" '
+    '"$http_user_agent" "$http_x_forwarded_for"';
+  access_log   /dev/stdout  main;
+  sendfile     on;
+  tcp_nopush   on;
+  server_names_hash_bucket_size 128; # this seems to be required for some vhosts
+
+  map $http_upgrade $connection_upgrade {
+    default upgrade;
+    ''      "";
+  }
+
+  server {
+    listen 80 default_server;
+    server_name _; 
+
+    location / {
+      proxy_http_version 1.1;
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection $connection_upgrade;
+    
+      proxy_set_header Host $host;
+      proxy_set_header X-Real-IP $remote_addr;
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header X-Forwarded-Proto $scheme;
+    
+      proxy_pass http://vaultwarden.zeabur.internal:80;
+    }
+  }
+}
+```
+In the file content above, `vaultwarden.zeabur.internal` in the `proxy_pass` block is the service name of Vaultwarden. Here, we used [Private Network](/deploy/private-networking) feature, which allows to access services through a fixed host name using an internal network.
+
+In the file above, we added a custom log format and log storage location, in addition to several custom nginx optimization parameters.
```

Comment:
```suggestion
```

I think this is unnecessary because it's derived from the default NGINX configuration of Zeabur.